tests/core/test_schemas.py

```diff
@@ -1283,6 +1284,48 @@ def test_lazy_dataframe_validation_nullable_with_checks() -> None:
         )
 
 
+def test_schema_empty():
+    """Ensure that an empty dataframe works for all valid pandera dtypes."""
+    # make sure all subclasses of pandera.dtypes.DataType are instantiated
+    import pandera.dtypes  # pylint: disable=C0415:
+    import pandera.engines.pandas_engine  # pylint: disable=C0415:
+    import pandera.engines.numpy_engine  # pylint: disable=C0415:
+
+    # find all subclasses
+    def get_subclasses(parent: type):
+        yield parent
+        for child in parent.__subclasses__():
+            yield from get_subclasses(child)
+
+    # create a valid schema of all possible dtypes pandera supports
+    skip = {
+        # these are abstract and should be excluded from column creation
+        pandera.dtypes.DataType,
+        pandera.engines.numpy_engine.DataType,
+        pandera.engines.pandas_engine.DataType,
+        # these fail when trying to make columns from them, but probably shouldn't
```

Comment:
I'd like to remove all exceptions that aren't `*.Datatype`, might be tough for those that where `pandera.Column(...)` doesn't work though -- given that that's how `pandera.Column` is supposed to be used.